packages/cli/src/utils/directory.test.ts

```diff
@@ -0,0 +1,84 @@
+import path from "path"
+import { coreCMSDir, coreDir, tmpCMSDir, tmpCMSWebhookUrlsFile, tmpCustomizationsSrcDir, tmpDir, tmpStoreConfigFile, tmpThemesCustomizationsFile, userCMSDir, userSrcDir, userStoreConfigFile, userThemesFileDir } from "./directory"
+
+const pathsToMatch = (expected: string, desired: string) => {
+  const expectedResolved = path.resolve(expected)
+  const desiredResolved = path.resolve(desired)
+
+  return expectedResolved === desiredResolved
+}
+
+describe('directory with base path', () => {
+  describe('coreDir', () => {
+    it('is the fastsstoreDir + core', () => {
```

Comment:
@gvc typo: `fastsstoreDir` should be `faststoreDir`

Reply:
the eyes of an eagle! Thank you!